src/content/zh-cn/updates/2017/10/remove-shadow-piercing.md

```diff
@@ -0,0 +1,43 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: 向shadow-piercing CSS选择器说再见。
+
+{# wf_updated_on: 2017-10-24 #}
+{# wf_published_on: 2017-10-24 #}
+{# wf_tags: webcomponents,shadowdom,style,css,deprecations,removals #}
+{# wf_featured_image: /web/updates/images/generic/styles.png #}
+{# wf_featured_snippet: 向shadow-piercing CSS选择器说再见。 #}
+{# wf_blink_components: Blink>DOM #}
+
+# Chrome 63将删除::shadow 和 /deep/ {: .page-title }
+
+{% include "web/_shared/contributors/robdodson.html" %}
+
+Translated by
+{% include "web/_shared/contributors/henrylim.html" %}
+
+在Chrome63之后，你将无法使用shadow-piercing 选择器， `::shadow` 和 `/deep/` 来设计(style) shadow root里面的内容。
+
+- `/deep/` 组合者(combinator)将会成为后代选择器。 `x-foo /deep/ div` 将会当成 `x-foo div`.
+- `::shadow` 伪元素(pseudo-element)将不会匹配任何元素。
+
+注：如果你的網頁使用Polymer, Polymer團隊已發布了一個[如何讓你迁移](https://www.polymer-project.org/blog/2017-10-18-upcoming-changes.html) `::shadow` 和 `/deep/`的文章。
+
+## 决定删除::shadow 和 /deep/ 的原因
+
+`::shadow` 和 `/deep/` 已经在Chrome 45被弃用了。这是在[2015年4月由Web组件聚会](https://www.w3.org/wiki/Webapps/WebComponentsApril2015Meeting)中的全部的出席者决定的。
+
+目前，shadow-piercing选择器的问题是它违反了封装(encapsulation)以及在某些时候组件无法更改组件的内部实施。
+
+注：目前JavaScript API如`querySelector()` 和`querySelectorAll()`会继续支持`::shadow` 和`/deep/`。目前这API的支持[正在在GitHub上讨论](https://github.com/w3c/webcomponents/issues/78)。
```

Comment:
[Callouts](https://developers.google.com/web/resources/markdown-syntax#callouts) like `Note:` should NOT be translated, they're handled differently by the markdown processor.